tests/Functional/ServiceInstantiationTest.php

```diff
@@ -134,7 +134,8 @@ protected static function bootKernel(array $options = []): KernelInterface
         $dispatcher = static::$kernel->getContainer()->get('event_dispatcher');
 
         $class = (Kernel::MAJOR_VERSION >= 5) ? RequestEvent::class : GetResponseEvent::class;
-        $event = new $class(static::$kernel, SymfonyRequest::create('/'), HttpKernelInterface::MASTER_REQUEST);
+        $requestType = (Kernel::MAJOR_VERSION >= 7) ? HttpKernelInterface::MAIN_REQUEST : HttpKernelInterface::MASTER_REQUEST;
```

Comment:
```suggestion
        $requestType = (Kernel::MAJOR_VERSION >= 6) ? HttpKernelInterface::MAIN_REQUEST : HttpKernelInterface::MASTER_REQUEST;
        
```
main was introduced in symfony 5.3, and master is deprecated since then. lets use it from symfony 6 on.